src/components/Nav/Mobile.js

```diff
@@ -11,6 +12,25 @@ import Emoji from "../Emoji"
 import { NavLink } from "../../components/SharedStyledComponents"
 import { translateMessageId } from "../../utils/translations"
 
+const Container = styled.div`
+  display: none;
```

Comment:
Adding `display: none` will hide all mobile elements from the a11y tree on desktop